src/cli/serve/integration_tests/reload_logging_config.test.ts

```diff
@@ -135,7 +135,7 @@ describe('Server logging configuration', function () {
           '--verbose',
         ]);
 
-        const message$ = Rx.fromEvent(child.stdout, 'data').pipe(
+        const message$ = Rx.fromEvent(child.stdout!, 'data').pipe(
```

Comment:
Nit: This is [nullable](https://nodejs.org/api/child_process.html#subprocessstdout). Given that its a test it should be okay though.